### PR TITLE
Improve code quality

### LINKED ISSUE #172: [PE-D] Not having sample pool for testing.
I was thinking that as the sample data was for testing purposes, not including sample data for pool list could be a feature flaw as it could hinders testing, and new users might not get to experience the application fully to determine whether or not they should use the application.

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: enhao25/ped#11

### PR DESCRIPTION
**Pull request by JoelHo**
Update JSON files for tests
Update equality checking for Passenger/Driver to include phone (i.e. can add 2 passengers as long as name and phone are both different)
Include Pools in sample data on first run
Update tags to better fit product (use department names)

### CODE REVIEW

**Review by lyueyang — APPROVED** [review]
LGTM

**Review by ZechariahTan — APPROVED** [review]
LGTM

File: src/test/java/seedu/address/model/person/driver/DriverTest.java [hunk 1]
```diff
@@ -33,12 +33,8 @@ public void isSameDriver() {
         // null -> returns false
         assertFalse(DRIVER_ALICE.isSameDriver(null));
 
-        // same name, all other attributes different -> returns true
```
Comment (ZechariahTan) [inline src/test/java/seedu/address/model/person/driver/DriverTest.java:36]: should an additional test case be added for if driver A and B has the same phone number but different names?

**Review by ZechariahTan — APPROVED** [review]
LGTM

**Pull request merged**